### ISSUE TITLE
Fix mislabeling of proportions in the shiny app in the wiki for syn4939350

### ISSUE THREAD

**Issue by kdaily** [MEP-LINCS/MEP_LINCS#31]

**Comment by markdane** [comment]
The issue is that we are displaying logit values and labeling them as proportions. I propose that in this release we leave the values as logits and change the labels in the CuratedFeatures file to reflect this. 

I would like to generalize the CuratedFeatures file but I believe this will break any code that is using it such as the shiny app. Likely at the same time, I'd like to add the proportion values (in addition to their logit values) to the data as these are easy to interpret.

**Comment by kdaily** [comment]
OK - I'll close this for now, and open an issue in the data explorer repo.
